workshops/geoserver/intro/doc/source/install/geoserver.rst

```diff
@@ -0,0 +1,213 @@
+
+    .. figure:: img/download_website.png
+       :width: 100%
+       
+       GeoServer website
+
+
+#.  From the `latest <http://geoserver.org/release/stable/>`__ release download the package appropriate to your platform.
+    
+    .. figure:: img/download_geoserver.png
+       :width: 100%
+       
+       Latest GeoServer download
+
+#. **Double-click** the file :file:`GeoServer-<version>.exe` to begin the installation.
+
+   .. note:: On some systems, you might see a Windows Security warning. OSGeo now signs the installer and executable (so it should not happen anymore).
+
+#. At the *Welcome* screen, click :guilabel:`Next`.
+
+   .. figure:: img/install_geoserver.png
+      :width: 75%
+
+      GeoServer installation Welcome screen
+
+#. Read the *License Agreement*, then click :guilabel:`I Agree`.
+
+   .. figure:: img/install_license.png
+      :width: 75%
+      
+      License Agreement
+
+#. Select the *Destination Folder* where you would like to install, and click :guilabel:`Next`.
+
+   .. figure:: img/install_directory.png
+      :width: 75%
+      
+      Destination folder for the installation
+
+#. Select the name and location of the *Start Menu Folder* that will be created for the Suite components, and click :guilabel:`Next`.
+
+   .. figure:: img/install_startmenu.png
+      :width: 75%
+      
+      Start Menu Folder to be created for the installation
+
+#. Choose the location of your Java Runtime Environment.
+
+   .. figure:: img/install_jre.png
+      :width: 75%
+      
+      Java Runtime path selection
+
+#. Use the default data directory included with the application.
+
+   .. figure:: img/install_data_directory.png
+      :width: 75%
+      
+      GeoServer data directory
+      
+#. Use the default :kbd:`admin` / :kbd:`geoserver` credentials.
+
+   .. figure:: img/install_admin_password.png
+      :width: 75%
+      
+      GeoServer administrator credentials
+      
+#. Port :kbd:`8080`:
+    
+   .. figure:: img/install_port.png
+      :width: 75%
+      
+      Web server port
+      
+#. Install GeoServer as a windows service.
+   
+   .. figure:: img/install_service.png
+      :width: 75%
+      
+      Install as service
+      
+#. When you are ready, click :guilabel:`Install` to start the installation.
+
+   .. figure:: img/install_ready.png
+      :width: 75%
+      
+      Ready to install
+
+#. After installation, click :guilabel:`Finish`.
+
+   .. figure:: img/install_finish.png
+      :width: 75%
+      
+      GeoServer has been installed
+
+#. Confirm GeoServer is working by visiting http://localhost:8080/geoserver in your browser.
+   
+   .. figure:: img/install_test.png
+      :width: 100%
+      
+      GeoServer Web Administration page
+
+Starting and stopping services
+------------------------------
+
+GeoServer has been installed as a windows service and is now running by default.  You can start and stop this services through the Start Menu.

```

Comment:
Maybe just do a search/replace to make sure Windows is capitalized everywhere. It's the little things.

Also "this services".